src/Repository/CurriculumInventoryAcademicLevelRepository.php

```diff
@@ -78,15 +78,25 @@ protected function attachCriteriaToQueryBuilder(
         ?int $limit,
         ?int $offset
     ): void {
-        if (array_key_exists('sequenceBlocks', $criteria)) {
-            $ids = is_array($criteria['sequenceBlocks']) ? $criteria['sequenceBlocks'] : [$criteria['sequenceBlocks']];
-            $qb->join('x.sequenceBlocks', 'sb');
-            $qb->andWhere($qb->expr()->in('sb.id', ':sequenceBlocks'));
-            $qb->setParameter(':sequenceBlocks', $ids);
+        if (array_key_exists('startingSequenceBlocks', $criteria)) {
```

Comment:
I'd be pretty surprised if someone queried the API by this, but I like the thoroughness!

Reply:
:heavy_check_mark: done.